lib/lwt/lwt_js_events.ml

```diff
@@ -387,6 +387,36 @@ let pointerover ?use_capture ?passive target =
 let pointerup ?use_capture ?passive target =
   make_event Dom_html.Event.pointerup ?use_capture ?passive target
 
+let transition_evn =
```

Comment:
I should have been more explicit. I think we can get rid of all old browser specific support.